lib/src/constants.dart

```diff
@@ -20,6 +20,15 @@ const String nameKey = 'name';
 /// String key in pubspec.yaml for the transformers map.
 const String transformersKey = 'transformers';
 
+/// Packages that are typically only used for their binaries.
+const List<String> commonBinaryPackages = const [
+  'build_runner',
+  'build_test',
+  'build_web_compilers',
+  'built_value_generator',
```

Comment:
Glad this is `built_value_generator` and not `build_value_generator` like in the PR description.

Reply:
Lol. Oops.